README.md

```diff
@@ -89,14 +89,17 @@ response_generator = client.start_with_media(media_generator=media_generator,
                                              response_types=response_types)
 ```
 
+_Compatibility note: the method `start_stream` is considered deprecated as of version 0.9.0, and is superceded by `start_with_media`, to explicitly denote whether a media generator is to be provided or not. 
```

Comment:
As I wrote in another comment, I think `start_stream` is perfectly fine and there's no need for this.
It's not a good enough reason to start messing with deprecation and interface changes... before we even went to real prod :)
Moreover, `start_with_media` sounds strange, I think it's "default enough" that you stream media to an ASR service - so no need to specify it so explicitly.

Reply:
Removed the compatibility note.